src/pudl/extract/ferc1.py

```diff
@@ -145,7 +145,11 @@
         "dbf": "f1_324_elc_expns",
         "xbrl": "electric_operations_and_maintenance_expenses_320",
     },
-    "electric_energy_account_ferc1": {
+    "electric_energy_account_sources_ferc1": {
+        "dbf": "f1_elctrc_erg_acct",
+        "xbrl": "electric_energy_account_401a",
+    },
+    "electric_energy_account_dispositions_ferc1": {
         "dbf": "f1_elctrc_erg_acct",
         "xbrl": "electric_energy_account_401a",
     },
```

Comment:
one xbrl and one dbf table are going to end up as two separate pudl tables. I simply added two instances of the dbf/xbrl tables with different pudl tables here in this extract dict. This means these raw tables will be extracted twice somewhat unnecessarily. but this seem more ideal than adding a fancy logic that let us know when to use these raw tables in two separate end-tables